src/js/utils/api.js

```diff
@@ -66,20 +77,40 @@ export default class api {
    *   { updates } updates Object
    * @return {Promise}
    */
-  static put(requestPath, { id, updates }) {
+  static put(requestPath, { id, updates } = {}) {
```

Comment:
Expected ')' and instead saw 'static'.<br>Missing semicolon.<br>'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Bad assignment.<br>Expected ')' and instead saw '{'.<br>Unexpected ')'.<br>Unexpected '{'.